Win32/Prj/wpcap.vcxproj

```diff
@@ -113,7 +113,8 @@
       <AdditionalDependencies>ws2_32.lib;..\..\..\..\packetWin7\Dll\Project\Release No NetMon and AirPcap\Packet.lib;%(AdditionalDependencies)</AdditionalDependencies>
     </Link>
     <PreBuildEvent>
-      <Command>win_flex -Ppcap_ -7 --outfile=..\..\scanner.c --header-file=..\..\scanner.h ..\..\scanner.l
+      <Command>call ..\..\..\GenVersion.bat
```

Comment:
You need to add `GenVersion.bat` to the commit; `git add` it and then do `git commit --amend` and push the results up to GitHub (or whatever the equivalent is if you're not using the Git command-line tools).